pandas/tests/indexes/datetimes/test_datetime.py

```diff
@@ -0,0 +1,1919 @@
+import warnings
+import numpy as np
+from datetime import date, timedelta, time
+
+import pandas as pd
+import pandas.util.testing as tm
+import pandas.compat as compat
+from pandas.compat import lrange
+from pandas.tslib import OutOfBoundsDatetime
+from pandas.core.common import PerformanceWarning
+from pandas.compat.numpy import np_datetime64_compat
+from pandas import (notnull, DatetimeIndex, Int64Index, Index, date_range,
+                    bdate_range, Series, DataFrame, Timestamp, datetime,
+                    offsets, NaT, _np_version_under1p8)
+
+from pandas.util.testing import assert_series_equal, assert_almost_equal
+
+randn = np.random.randn
+
+
+class TestDatetimeIndex(tm.TestCase):
+    _multiprocess_can_split_ = True
+
+    def test_construction_with_alt(self):
```

Comment:
so ok with splitting this up a bit now. e.g. test_construction (and when you do, copy the class (name), and leave the methods alone (even though they say ``test_construction_*`` for now that's ok))